local_env_samples/all_model_training_sample.py

```diff
@@ -32,14 +34,13 @@ def read_by_line():
 
 # Save training dataset (streamed)
 dataset_response: SaveDatasetResponse = sdk.save_dataset(read_by_line(),
-                                                         "sample-dataset",
-                                                         overwrite=True)
+                                                         "sample-dataset")
 
 # Train new models
 train_request_bodies = []
 for model in models_to_train:
     train_request_bodies.append(
-        TrainRequestBody(model=model, datasets="sample-dataset.jsonl", no_cache=False, GPU=False))
+        TrainRequestBody(model=model, dataset_id=dataset_response.dataset_id, no_cache=False, gpu=False))
```

Comment:
Similar to `SaveDatasetResponse` I'm seeing:
<img width="639" alt="Screen Shot 2023-10-26 at 9 22 29 AM" src="https://github.com/graphgrid/graphgrid-sdk-python-examples/assets/6307559/3ea7a698-8deb-4add-a01b-b97dd06e1b6e">
For 
```
@dataclass
class TrainRequestBody:
    """Store Airflow configuration json/request bodies"""
    model: NlpModel
    datasetId: str # <--- this here
```

Reply:
nvm, resolved